recipe.yaml

```diff
@@ -1,7 +1,7 @@
 ---
 RecipeFormatVersion: '2020-01-25'
 ComponentName: aws.greengrass.labs.database.InfluxDB
-ComponentVersion: '1.0.0'
+ComponentVersion: '1.1.0'
```

Comment:
Because this is backwards incompatible, should we call it 2.0.0 instead of just minor version bump.

Reply:
Fixed